spec/monthday.html

```diff
@@ -294,6 +298,22 @@ <h1>ValidateMonthDay ( _month_, _day_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-rejectmonthday" aoid="RejectMonthDay">
+      <h1>RejectMonthDay ( _month_, _day_ )</h1>
+      <emu-alg>
+        1. Assert: _month_ and _day_ are integer Number values.
+        1. If ! ValidateMonthDay(_month_, _day_) is *false*, then
+          1. Throw a *RangeError* exception.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-constrainmonthday" aoid="ConstrainMonthDay">
+      <h1>ConstrainMonthDay ( _month_, _day_ )</h1>
+      <emu-alg>
+        1. Assert: _month_ and _day_ are integer Number values.
+        1. Return ! ConstrainDate(1972, _month_, _day_).
```

Comment:
Wouldn't this return a `Date` record? And we return it directly in `RegulateMonthDay` too. Maybe we should remove the `[[Year]]` field from this record?

Reply:
Ah, we do something similar down below in `BalanceMonthDay` could you please do that instead?